tools/devops/device-tests.yml

```diff
@@ -0,0 +1,236 @@
+resources:
```

Comment:
I think we should put this file in tools/devops too.

Reply:
What do you mean? `device-tests.yml` is in `tools/devops`. Are you talking about moving the `repositories` section into its own file so it can be re-used? I don't know if that works (:

Reply:
Never mind, for some reason I thought you'd put this file in jenkins/. My bad.